clients/python-wrapper/pyproject.toml

```
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lakefs"
+version = "0.14.0"
+description = "lakeFS Python SDK Wrapper"
+keywords = ["OpenAPI", "OpenAPI-Generator", "lakeFS API", "Python Wrapper"]
+requires-python = ">=3.9"
+readme = { file = "README.md", content-type = "text/markdown" }
```

`content-type` is not strictly required, as it is auto-detected based on extension.

```suggestion
readme = "README.md"
```